index.html

```diff
@@ -10,8 +10,8 @@
     <h1>Welcom to the Background Generator</h1>
     <input type="color" class="color-1">
     <input type="color" class="color-2">
-    <h2>Current CSS Background</h2>
-    <h3></h3>
+    <h2>This is the background</h2>
```

Comment:
hmm looks good.

(note the person who made this branch is also the one oversee this project. :P)

Reply:
yay, it is good, i completely agree

Reply:
anyway this is comment on line 13